src/main/java/com/epam/ta/reportportal/config/DataStoreConfiguration.java

```diff
@@ -79,4 +85,15 @@ public Thumbnailator userPhotoThumbnailator(@Value("${datastore.thumbnail.avatar
 	public ContentTypeResolver contentTypeResolver() {
 		return new TikaContentTypeResolver();
 	}
+
+	private void dataStorePostInit(DataStore dataStore) {
+		try {
+			dataStore.load(integrationSaltPath);
+		} catch (Exception ex) {
```

Comment:
Let's probably specify concrete exceptions "file not found" or smth here

Reply:
done